config/config.go

```diff
@@ -165,6 +165,33 @@ type Authorization struct {
 
 	// PolicyErrRetryInterval represent the retry interval when fail to get the policies from Athenz server.
 	PolicyErrRetryInterval string `yaml:"policyErrRetryInterval"`
+
+	// Access represents the configuration to control access token verification.
+	Access []Access `yaml:"access_tokens"`
+
+	// Role represents the configuration to control role token verification.
+	Role Role `yaml:"role_token"`
```

Comment:
Not `roletoken`?
In the client-sidecar it was `roletoken`.

Reply:
fixed: 3fb1a1bdad45417c36b080caa5d1f734b0e3a7dd

keep the bad naming for consistency, resolve the naming convention in